src/main/java/racing/InputView.java

```diff
@@ -0,0 +1,31 @@
+package racing;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final int INPUT_THRESHOLD = 2;
+    public static int car() {
+        System.out.println("자동차 대수는 몇 대 인가요? (2 이상의 자연수 입력)");
+        Scanner scanner = new Scanner(System.in);
+        int car_count = scanner.nextInt();
+        isCorrectInput(car_count);
+
+        return car_count;
+    }
+
+    public static int move() {
+        System.out.println("시도할 회수는 몇 회 인가요? (2 이상의 자연수 입력)");
+        Scanner scanner = new Scanner(System.in);
+        int moveCount = scanner.nextInt();
+        isCorrectInput(moveCount);
+
+        return moveCount;
+    }
+
+    public static boolean isCorrectInput(int input) {
+        if (input < INPUT_THRESHOLD) {
+            throw new IllegalArgumentException("PLEASE INPUT APPROPRIATE NUMBER (OVER 1)");
+        }
+        return true;
+    }
```

Comment:
```suggestion
        validate(moveCount);

        return moveCount;
    }

    public static void validate(int carCount) {
        if (input < INPUT_THRESHOLD) {
            throw new IllegalArgumentException("PLEASE INPUT APPROPRIATE NUMBER (OVER 1)");
        }
    }
```

boolean 값을 리턴했지만 사용하지 않네요
단순히 유효성 검증만을 위한 메서드라면 isXXX 보다는 
validateXXX 나 checkXXX 같은 메서드 이름을 사용하는 것이 좋습니다.